src/components/TabSelect/TabSelect.js

```diff
@@ -104,8 +104,13 @@ TabSelect.propTypes = {
   /**
    * Children of `TabSelect` must be a `Tab` component
    */
+  // eslint-disable-next-line
```

Comment:
Why? I think it's always good to go with `// eslint-disable name-of-specific-rule` to make that more clear..

Reply:
agreed, added the reasoning also